examples/8a_benchmark_write1D2D_parallel.cpp

```diff
@@ -0,0 +1,682 @@
+/* Copyright 2020 Junmin Gu, Axel Huebl
+ *
+ * This file is part of openPMD-api.
+ *
+ * openPMD-api is free software: you can redistribute it and/or modify
+ * it under the terms of of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * openPMD-api is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with openPMD-api.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <openPMD/openPMD.hpp>
+#include "openPMD/auxiliary/Environment.hpp"
+
+#include <mpi.h>
+
+#include <iostream>
+#include <memory>
+#include <random>
+#include <vector>
+#include <fstream>
+
+#if openPMD_HAVE_ADIOS2
+#   include <adios2.h>
```

Comment:
Hm, if we are using ADIOS2 here directly we should expose it in the build system...

It's ok to quick-hack this, but why do we do this here? Is this for validation or direct comparison?

Reply:
This should be related with the direct adios2 test. And will remove those and no need for this.